subsys/bluetooth/services/fast_pair/include/fp_common.h

```diff
@@ -19,7 +19,15 @@ extern "C" {
 #endif
 
 /** UUID of the Fast Pair GATT Service. */
-#define BT_FAST_PAIR_SERVICE_UUID  0xFE2C
+#define BT_FAST_PAIR_SERVICE_UUID	0xFE2C
```

Comment:
Since it's internal header, maybe we could name it `FP_SERVICE_UUID`? Thanks to that we will use common prefix in this file.